mrc-ide/wodin-deploy
# add basic wodin deploy

This PR will add very basic functionality to the wodin deploy tool. I have included a couple of comments in the diff to explain the changes. This will add functionality to deploy on localhost with all the website configurations.

To test:
Add a breakpoint in the `test_constellation` file in the `test_start_and_stop` test before `obj.stop(...)`, run the test and when the python program pauses the website should be available on localhost.

Still to do:
* add functionality to deploy with SSL cert and key to epimodels.dide.ic.ac.uk (will involve also reading the vault config)
* CLI tool
* update sites functionality (way to update all the sites without tearing down all containers and restarting everything)

Note: I will be making tickets for the future PRs soon but they will follow the bullet points above!
